tests/state_test.go

```diff
@@ -29,8 +29,9 @@ import (
 )
 
 var (
-	baseDir      = filepath.Join(".", "testdata")
-	stateTestDir = filepath.Join(baseDir, "GeneralStateTests")
+	eipTests          = filepath.Join(".", "testdata", "EIPTests", "StateTests")
+	generalStateTests = filepath.Join(".", "testdata", "GeneralStateTests")
+	execSpecTests     = filepath.Join(".", "execution-spec-tests", "fixtures", "state_tests")
```

Comment:
nit: it would be better to define a slice here and use it below. So if in the future directories need to be modified, there is only a single place to change.